controllers/datafoundation.go

```diff
@@ -144,9 +156,31 @@ func (r *dataFoundationReconciler) initReconciler(reconciler *ManagedFusionOffer
 
 	r.ocsInitialization.Name = "ocsinit"
 	r.ocsInitialization.Namespace = offering.Namespace
+
+	r.availableCRDs = reconciler.AvailableCRDs
 }
 
 func (r *dataFoundationReconciler) reconcilePhases() (ctrl.Result, error) {
+	// Checking for CRDs that are installed after the agent was started and add watchers for the new CRDs
```

Comment:
We are not adding any watches right now.
```suggestion
	// Checking for CRDs that were installed after the agent was started
```